oneview_redfish_toolkit/client_session.py

```diff
@@ -115,6 +115,14 @@ def get_session_ids():
     return [sess_dict['session_id'] for _, sess_dict in session_map_items]
 
 
+def get_session_by_token(token):
```

Comment:
Why not use existing method  def _get_session_id_by_token(token)? Maybe rename existing to get_session_id_by_token(token).